packages/angular-cli/commands/build.ts

```diff
@@ -42,6 +43,8 @@ const BuildCommand = Command.extend({
       }
     }
 
+    Version.assertPostWebpackVersion();
```

Comment:
This should not be only on builds, but on calls to `ng`. Testing and generators also have changed and assume post-webpack.